app/components/marketing/events-section.tsx

```diff
@@ -64,7 +60,7 @@ export function EventCard({ event, index }: { event: ExtendedEvent; index: numbe
         )}
 
         <div>
-          <div className="font-bold text-white line-clamp-1">{event.group.name}</div>
+          <div className="font-bold text-white line-clamp-1">{event.group?.name}</div>
```

Comment:
We can account for when `group` is `undefined`. We can either conditionally render the group name our update or types so that `group` is required (assuming it works wherever `FullEvent` is used).